README.md

```diff
@@ -88,3 +88,14 @@ If the block number is specified, but no EVM version is specified, it is automat
 ```bash
 ape plugins install alchemy
 ```
+
+## Remote Anvil Node
+
+To connect to a remote anvil node, set up your config like this:
+
+```yaml
+foundry:
+  host: https://anvil.example.com
+```
+
```

Comment:
How can I use provider settings dynamically instead of via config?

Reply:
you can do python

```python
with networks.ethereum.local.use_provider("foundry", provider_settings={"host": "1222.22.22.2:6555"}):
```